projects/go-lib/src/lib/components/go-select/go-select.component.html

```diff
@@ -31,27 +33,32 @@
   [typeToSearchText]="typeToSearchText"
   [virtualScroll]="virtualScroll"
   (scrollToEnd)="onScrollToEnd()"
-  (scroll)="onScroll($event)">
-  <ng-template
+  (close)="onClose()"
+  (scroll)="onScroll($event)"
+  #select
+  >
+    <ng-template
     *ngIf="multiple && items?.length"
     ng-header-tmp>
     <button
       type="button"
       (click)="onSelectAll()"
       class="go-select__select-all-button"
       [ngClass]="{ 'go-select__select-all-button--dark' : theme === 'dark' }"
-      *ngIf="showSelectAll && control.value?.length < items?.length">
-      Select All
+      *ngIf="showSelectAll && (control.value?.length < items?.length || typeahead)"
```

Comment:
> As mentioned in the `.component.ts` file, make sure to update the naming of `typeAhead` in the html as well.

Hi @swimtech20 ```typeahead``` was declared before this PR. Changing it to ```typeAhead``` would be a breaking change. Although we can do something like this ```@Input('typeahead') typeAhead``` but it would be hard to maintain since ng-select accepts ```typeahead``` it would be easier to understand. What do you think ?

Reply:
Hi @vishal-tangoe. I see what you mean, because it's referenced in the mobi app as well, that the breaking change. This is fine then!